plugins/container/go-worker/pkg/container/cri.go

```diff
@@ -520,20 +518,21 @@ func (c *criEngine) Listen(ctx context.Context, wg *sync.WaitGroup) (<-chan even
 				}
 
 				var info event.Info
-				// verbose true to return container.Info
-				ctr, err := c.client.ContainerStatus(ctx, evt.ContainerId, true)
+				returnInfo := true
+				ctr, err := c.client.ContainerStatus(ctx, evt.ContainerId, returnInfo)
 				if err != nil || ctr == nil {
 					info = event.Info{
 						Container: event.Container{
 							Type:        c.runtime,
 							ID:          shortContainerID(evt.ContainerId),
 							FullID:      evt.ContainerId,
 							CreatedTime: nanoSecondsToUnix(evt.CreatedAt),
+							IsPodSandbox: true,
 						},
 					}
 				} else {
 					cPodSandbox := evt.GetPodSandboxStatus()
-					podSandboxStatus, _ := c.client.PodSandboxStatus(ctx, cPodSandbox.GetId(), false)
+					podSandboxStatus, _ := c.client.PodSandboxStatus(ctx, cPodSandbox.GetId(), returnInfo)
```

Comment:
For posterity, `cPodSandbox` and `podSandboxStatus` are available also when it's a sandbox, but we won't use that data for that case.